konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/step/ml/classifier/ClassifierOutputRunner.java

```
@@ -0,0 +1,201 @@
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+
+package ai.konduit.serving.pipeline.impl.step.ml.classifier;
+
+import ai.konduit.serving.annotation.runner.CanRun;
+import ai.konduit.serving.pipeline.api.context.Context;
+import ai.konduit.serving.pipeline.api.data.Data;
+import ai.konduit.serving.pipeline.api.data.NDArray;
+import ai.konduit.serving.pipeline.api.data.ValueType;
+import ai.konduit.serving.pipeline.api.step.PipelineStep;
+import ai.konduit.serving.pipeline.api.step.PipelineStepRunner;
+import ai.konduit.serving.pipeline.util.DataUtils;
+import lombok.AllArgsConstructor;
+
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.DoubleStream;
+
+@AllArgsConstructor
+@CanRun(ClassifierOutputStep.class)
+public class ClassifierOutputRunner implements PipelineStepRunner {
+
+    protected final ClassifierOutputStep step;
+
+
+    @Override
+    public void close() {
+
+    }
+
+    @Override
+    public PipelineStep getPipelineStep() {
+        return step;
+    }
+
+    @Override
+    public Data exec(Context ctx, Data data) {
+
+        String inputName = step.inputName();
+
+
+        if (inputName == null) {
+            String errMultipleKeys = "NDArray field name was not provided and could not be inferred: multiple NDArray fields exist: %s and %s";
+            String errNoKeys = "NDArray field name was not provided and could not be inferred: no image NDArray exist";
+            inputName = DataUtils.inferField(data, ValueType.NDARRAY, false, errMultipleKeys, errNoKeys);
+        }
+
+
+        NDArray classifierOutput = data.getNDArray(inputName);
+
+        boolean batch = false;
+        if (classifierOutput.shape().length == 2 && classifierOutput.shape()[0] > 1) {
+            batch = true;
+        }
```

Let's check that rank (shape().length) is 1 or 2. For 3, we should throw an exception like:
```
"Invalid input to ClassifierOutputStep: only rank 1 or 2 inputs are available, got array with shape " + Arrays.toString(classifierOutput().shape())
```